src/Reader.tsx

```diff
@@ -132,6 +197,21 @@ export function Reader({
     });
   }
 
-  if (!template) throw new Error('Template is not set');
-  return <View template={template} width={width} height={height} {...rest} />;
+  if (!templateUrl || !allowedUris) {
```

Comment:
`allowedUris` is null here, so it will always throw `Template is not set`

Reply:
yep🤦‍♂️
when at all times i should set allowedUrls to be equal to jzip url and epubjs url since we are storing that in the file system

 I'm not setting allowedUrls when !isExternalSource case is true, it should be equal to the src url plus the jzip and epubjs thingy and we should modify the allowedUrls there.

and in the case where the src is external and we are not downloading the file we do not need to modify the allowedUrls.

and in the case where the src is external and we are downloading the file I have already added the case where i set allowedUrls with downloaded book url and other two js files. this was the only case i had it set.

is this seems ok @shlomizadok ?

Reply:
Seems great @DimuthuWeerathunga (And thank you so much for making this awesome change ❤️)